CMakeLists.txt

```diff
@@ -36,8 +36,8 @@ include(simd)
 ################################ Configurations #######################################
 option(WITH_GPU         "Compile PaddlePaddle with NVIDIA GPU"          ${CUDA_FOUND})
 option(WITH_AVX         "Compile PaddlePaddle with AVX intrinsics"      ${AVX_FOUND})
```

Comment:
Please also consider Mac OS. currently, MKLDNN doesn't support Mac OS X, it needs to be disabled under Mac OS X. https://github.com/01org/mkl-dnn/issues/91